<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacChanger</class>
 <widget class="QMainWindow" name="MacChanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mac Changer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="macAddressInput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>xx:xx:xx:xx:xx:xx</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>wlan0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout_Me">
    <property name="title">
     <string>About Me</string>
    </property>
    <addaction name="actionAbout_me"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuAbout_Me"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_me">
   <property name="text">
    <string>About me</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
